files/es/web/security/same-origin_policy/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Política Same-origin
+slug: Web/Security/Same-origin_policy
+tags:
+  - CORS
+  - JavaScript
+  - Mismo-Origen
+  - Política Same-Origin
+  - Seguridad
+translation_of: Web/Security/Same-origin_policy
+original_slug: Web/Security/Same-origin_politica
+---
+La política same-origin (mismo-origen) restringe cómo un documento o script cargado desde un origen puede interactuar con un recurso de otro origen. Es un mecanismo de seguridad crítico para aislar documentos potencialmente maliciosos.
+
+## Definición de origen
+
+Dos páginas tienen el _mismo origen_ si el {{Glossary("protocol","protocolo")}}, {{Glossary("port","puerto")}} (si es especificado) y {{Glossary("host","anfitrión")}} son los mismos para ambas páginas. Verá esto a veces referido como la "tupla esquema/anfitrión/puerto" (donde una "tupla" es un conjunto de componentes que juntos forman un todo).
+
+La siguiente tabla muestra ejemplos de comparaciones de origenes para la URL `http://store.company.com/dir/page.html`:
+
+| URL                                               | Resultado    | Razón                |
+| ------------------------------------------------- | ------------ | -------------------- |
+| `http://store.company.com/dir2/other.html`        | Mismo origen | Solo la ruta difiere |
+| `http://store.company.com/dir/inner/another.html` | Mismo origen | Solo la ruta difiere |
+| `https://store.company.com/secure.html`           | Fallo        | Diferente protocolo  |
+| `http://store.company.com:81/dir/etc.html`        | Fallo        | Diferente puerto     |
+| `http://news.company.com/dir/other.html`          | Fallo        | Diferente host       |
+
+Ver también [definición de origen para `file:` URLs](/es/docs/Same-origin_policy_for_file:_URIs), puesto que su comparación es más complicada.
+
+### Orígenes heredados
+
+Los scripts ejecutados desde páginas con una URL `about:blank` o `javascript:` heredan el origen del documento que contiene esa URL, puesto que esos tipos de URLs no contienen información sobre un servidor de origen.
+
+> **Nota:** Por ejemplo, `about:blank` a menudo se usa como URL de nuevas ventanas popup en las que el script padre escribe contenido (por ejemplo mediante el mecanismo {{domxref("Window.open()")}}). Si este popup además contiene JavaScript, ese escript heredará el mismo origen que el script que lo ha creado.
+
+> **Advertencia:** `data:` URLs obtienen un nuevo, vacío, contexto de seguridad.
+
+### Excepciones en Internet Explorer
+
+Internet Explorer tiene dos excepciones mayores en lo que se refiere a la política same-origin
+
+- Zonas de Confianza: si ambos dominios pertenecen a una zona de alta confianza e.g, dominios corporativos, entonces las limitaciones del mismo origen no son aplicadas.
+- Puerto: IE no incluye puerto en los componentes de Same Origin, por lo tanto http\://company.com:81/index.html y http\://company.com/index.html se consideran del mismo origen y no se aplican restricciones.
```

Comment:
```suggestion
- Puerto: IE no incluye puerto en los componentes de Same Origin, por lo tanto <http://company.com:81/index.html> y <http://company.com/index.html> se consideran del mismo origen y no se aplican restricciones.
```

Reply:
For fake/example links, I thin they tend to use backticks so they are wrapped in a code fence